portfolio/src/main/webapp/script.js

```diff
@@ -153,7 +153,7 @@ async function start(page) {
     }
     else {
       document.getElementById("comment-section").innerText = "Please log in to view comment section";
-      document.getElementById("set-nickname").style.display = "none";
+      document.getElementById("nickname").style.display = "none";
```

Comment:
tiny nit, the standard style is to use  should be using single ' instead of double quotes " in JavaScript.  I know it's annoying because I believe the HTML style is double quotes, and Java requires double quotes too..

Reply:
got it! fixed